permissions.go

```diff
@@ -368,6 +368,33 @@ func (jobState *JobState) getPermissionsForRunStep(step Step) ([]string, error)
 		}
 	}
 
+	// Dependabot auto approve
+	if step.Env["GITHUB_TOKEN"] != "" && isGitHubToken(step.Env["GITHUB_TOKEN"]) {
+		if strings.Contains(runStep, "gh pr review --approve") {
+			permissions = append(permissions, "pull-requests: write")
```

Comment:
@Devils-Knight can you please move all references of permissions, e.g. `pull-requests: write` to be a constant? That will help avoid typos in the future.